components/BridgeContent/SocketTxHistory/AssetAmount.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { memo, useMemo } from 'react';
+
+import { Box, Typography } from '@mui/material';
+import { Asset } from 'types/Bridge';
+import Image from 'next/image';
+import formatNumber from 'utils/formatNumber';
+import { formatUnits } from 'viem';
+import * as chains from 'wagmi/chains';
+
+type Props = { asset: Asset; amount: number; mobile?: boolean };
+
+const AssetAmount = ({ asset, amount, mobile }: Props) => {
+  const chain = useMemo(() => Object.values(chains).find(({ id }) => id === asset.chainId), [asset.chainId]);
+  return (
+    <Box display={'flex'} flexDirection={'column'} gap={'11px'}>
+      <Box display={'flex'} alignItems={'center'} height={16} gap={'4px'}>
+        <Image
+          src={asset.logoURI}
+          alt={asset.symbol}
+          width={mobile ? 24 : 16}
+          height={mobile ? 24 : 16}
+          style={{
+            maxWidth: '100%',
+            height: 'auto',
+          }}
+        />
+        <Typography
+          variant="body1"
+          fontSize={mobile ? '19px' : '14px'}
+          whiteSpace={'nowrap'}
+          fontWeight={mobile ? 500 : 600}
+        >
+          {formatNumber(formatUnits(BigInt(amount), asset.decimals))} {asset.symbol}
+        </Typography>
+      </Box>
+      <Box display={'flex'} alignItems={'center'} height={16} gap={'4px'}>
+        <Image
+          src={chain?.icon || ''}
```

Comment:
probably you already saw this, but compilation is failing due to `.icon` field not existing in the type `Chain` from `wagmi/chains`